cmd/minishift/cmd/util/util.go

```diff
@@ -50,3 +63,30 @@ func ExitIfNotRunning(driver drivers.Driver, machineName string) {
 		atexit.ExitWithMessage(0, fmt.Sprintf("The execution of this command requires a running '%s' VM, but there is currently none.", machineName))
 	}
 }
+
+// UnpackAddons will unpack the default addons into addons default dir
+func UnpackAddons(addonsDir string) error {
+	for _, asset := range DefaultAssets {
+		if err := bindata.RestoreAssets(addonsDir, asset); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func CreateUpdateMarker(markerPath string, data UpdateMarker) error {
```

Comment:
Not sure whether I would put this method here, instead of _update.go_. Feels you just do it, because of the test.